src/assets/variables.scss

```diff
@@ -1,20 +1,34 @@
 /*colors*/
 //blue
-$primary001: #7BB9FA;
-$primary002: #499FF8;
+$primary001: #7bb9fa;
+$primary002: #499ff8;
 $primary003: #d7eafd;
 //green
-$success001: #38DB81;
-$success002: #21B965;
-$success003: #86E9B2;
+$success001: #38db81;
+$success002: #21b965;
+$success003: #86e9b2;
 //red
-$danger001: #F67F7F;
-$danger002: #F34E4E;
-$danger003: #FCD9D9;
+$danger001: #f67f7f;
+$danger002: #f34e4e;
+$danger003: #fcd9d9;
 //grey
-$grey001: #D0D0D0;
-$grey002: #BFBFBF;
-$grey003: #F1F1F1;
+$grey001: #d0d0d0;
+$grey002: #bfbfbf;
+$grey003: #f1f1f1;
 //other
 $black: #282828;
-$white: #FFFFFF;
\ No newline at end of file
+$white: #ffffff;
+
+/*typography*/
+//font
+@import url("https://fonts.googleapis.com/css?family=Montserrat");
```

Comment:
It think we can create some file (for example `index.scss`) which could handle some global styles (for example we could apply `Montserrat` for the root element to apply it to all of the components) as well as global imports like that one. We could later import that file for example in `App.jsx`. We may also want to add `display=swap` or define used weights to limit the amount of fetched fonts - smth like `@import url("https://fonts.googleapis.com/css?family=Montserrat:wght@400;600;800&display=swap");`